ova/generate_ova.sh

```diff
@@ -138,22 +137,34 @@ main() {
             fi
             shift 2
         ;;
+        "-o" | "--opendistro")
+            if [ -n "$2" ]; then
 
-        "-e" | "--elastic-version")
+                export OVA_OPENDISTRO_VERSION="$2"
+                OPENDISTRO_VERSION="$2"
+                HAVE_OPENDISTRO_VERSION=true
```

Comment:
These variables should be unified. Also, it isn't necessary to export the variable.